source/includes/_prices.md

```diff
@@ -20,22 +22,128 @@ curl "http://api.lettuce.money/prices?to_symbol=ZAR&from_symbols=BTC,USD,MSFT"
   "fromSymbols": [
     {
       "fromSymbol": "BTC",
-      "lastUpdated": "2018-12-31T11:59:01.001Z",
-      "price": 0.000021,
-      "source": "Crypto Compare"
+      "price": 53504.54788657036,
+      "source": "cryptocompare"
+    },
```

Comment:
Would each of these records not require a timestamp?

Reply:
Right now they don't. They are as @kinggs requested them to look, in the first planning session. We never changed it. Also it would be redundant since the service response time is always the same as the provider response time